composer.json

```diff
@@ -52,6 +52,10 @@
         "sort-packages": true,
         "platform": {
             "php": "8.0.99"
+        },
+        "allow-plugins": {
```

Comment:
Eh, we should really leave these out :D 

I'll let it pass - will probably drop them again in future